monkey/monkey_island/cc/services/reporting/test_pth_report.py

```diff
@@ -1,59 +0,0 @@
-import uuid
-
-from monkey_island.cc.models import Monkey
-from monkey_island.cc.services.reporting.pth_report import PTHReportService
-
-
-class TestPTHReportServiceGenerateMapNodes():
```

Comment:
Make sure @mssalvatore is aware

Reply:
looks like a few tests were removed. What is the rationale?

Reply:
These tests were testing unused code (and possibly broken). PTH map was broken and was removed from report, though I have not tested if the code for "strong users" are also broken. Opened an issue for this #958. I don't want to spend time fixing that effectively does nothing.